### PR TITLE
Add From the Front

### PR DESCRIPTION
**Pull request by cedmax**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -108,7 +108,7 @@
 - [Ruby Taiwan](https://github.com/rubytaiwan/AMA) - Helping developers in Taiwan with any questions. Can ask in English or Chinese.
 - [DevOps Taiwan](https://github.com/DevOpsTW/AMA) - Helping DevOps engineers with any questions.
 - [Vim Taiwan](https://github.com/vim-tw/ama) - Helping vimmers in Taiwan with any questions.
-
+- [From The Front](https://github.com/fromthefront/ama) - No-profit organisation taking care of the homonymous Italian front end conference series.
```
Comment (sindresorhus) [inline readme.md:111]: What does `homonymous` mean in this context? Mind simplifying the description?

Comment (cedmax) [inline readme.md:111]: I meant that the conference series had the same name of the no-profit organisation. 
I'm not a native English speaker and I thought it was straight forward, if not how would you phrase it?

Comment (sindresorhus) [inline readme.md:111]: I don't know. Up to you. Just more succinct.

Comment (cedmax) [inline readme.md:111]: Well, it's not much longer than the average and "homonymous" can only have the meaning of "having the same name", in the given context. But if that's a big deal, given that it's your repo and you make the rules, I'll find an alternative.

**Comment by sindresorhus** [comment]
Next time don't open a new PR. Just update the existing one.

https://github.com/RichardLitt/docs/blob/master/amending-a-commit-guide.md

**Pull request closed without merging**